code/ecdsap256/Hacl.P256.fsti

```diff
@@ -147,7 +183,18 @@ val ecdsa_sign_p256_without_hash: result: lbuffer uint8 (size 64)
   )
 
 
-[@ (Comment " The input of the function is considered to be public, 
+[@@ CPrologue "
+/****************/
+/* Verification */
+/****************/
+
+/*
+  The user MUST validate the public key before calling any of the
```

Comment:
The verification function already validates the key itself.

Reply:
changed